bids2openminds/main.py

```diff
@@ -12,6 +14,64 @@
 from .mapping import bids2openminds_instance
 
 
+def create_openminds_person(full_name):
+    # Regex for detecting any unwanted characters.
+    name_regex = re.compile(
```

Comment:
@apdavison I added this regex to verify for unwanted characters in names, can you please verify it?